### PR TITLE
Feat/in114 mock job posting objects

### PR DESCRIPTION
**Pull request by ReneReid**
#### User Story
IN-114: create mock JobPosting Objects
https://chaosneutral.atlassian.net/jira/software/projects/IN/boards/1/backlog?selectedIssue=IN-114

#### Changes made
created a mock job posting object for each category (listed below), stored in Internify -> client -> src -> models -> mockJobDataObjects.js

(1) front end developer
(2) back end developer
(3) full stack developer
(4) data scientist
(5) blank job object (all fields are blank)

UPDATE: fields of job posting objects have been aligned with survey conducted at beginning of term
Also, per Tommy's suggestion, the file containing the job objects has been renamed from "mockDataObjects.js" to
"templateDataObjects.js".  This is to reflect the fact that these objects are not temporary. Also casing changed per comments below. 

#### Does your new code introduce new warnings on dev console?
No.

#### Test Steps
Look inside repository: 
Internify -> client -> src -> models -> mockJobDataObjects.js

### CODE REVIEW

**Review by ghost — APPROVED** [review]
Excellent idea, having a blank Student, will make future story for template selection a lot easier.

File: client/src/models/mockJobDataObjects.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    name: "Insert Name",
+    email: "Insert Email",
+    linkedIn: "Insert LinkedIn",
+    other: "Insert Other Information",
+    applicationSteps: "Insert Application Steps",
+  },
+};
+
+
+const frontendStudent = {
```
Comment (tommytwm) [inline client/src/models/mockJobDataObjects.js]: change to camel casing: `frontEndStudent`

File: client/src/models/mockJobDataObjects.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
+const backendStudent = {
```
Comment (tommytwm) [inline client/src/models/mockJobDataObjects.js]: change to camel casing `backEndStudent`

File: client/src/models/mockJobDataObjects.js [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+    name: "Insert Name",
+    email: "Insert Email",
+    linkedIn: "Insert LinkedIn",
+    other: "Insert Other Information",
+    applicationSteps: "Insert Application Steps",
+  },
+};
+
+
+const datascienceStudent = {
```
Comment (tommytwm) [inline client/src/models/mockJobDataObjects.js]: `dataScienceStudent`

File: client/src/models/mockJobDataObjects.js [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+    name: "Insert Name",
+    email: "Insert Email",
+    linkedIn: "Insert LinkedIn",
+    other: "Insert Other Information",
+    applicationSteps: "Insert Application Steps",
+  },
+};
+
+
+const fullstackStudent = {
```
Comment (tommytwm) [inline client/src/models/mockJobDataObjects.js]: `fullStackStudent`

File: client/src/models/mockJobDataObjects.js [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+    applicationSteps: "",
+  },
+};
+
+export {
+  backendStudent,
+  frontendStudent,
+  datascienceStudent,
+  fullstackStudent,
+  blankStudent,
```
Comment (tommytwm) [inline client/src/models/mockJobDataObjects.js]: change to camel casing

**Review by tommytwm — CHANGES_REQUESTED** [review]

File: client/src/models/mockJobDataObjects.js (hunk 1 above)
Comment (ReneReid) [reply client/src/models/mockJobDataObjects.js]: changed

File: client/src/models/mockJobDataObjects.js (hunk 2 above)
Comment (ReneReid) [reply client/src/models/mockJobDataObjects.js]: changed

File: client/src/models/mockJobDataObjects.js (hunk 3 above)
Comment (ReneReid) [reply client/src/models/mockJobDataObjects.js]: changed

File: client/src/models/mockJobDataObjects.js (hunk 4 above)
Comment (ReneReid) [reply client/src/models/mockJobDataObjects.js]: changed

File: client/src/models/mockJobDataObjects.js (hunk 5 above)
Comment (ReneReid) [reply client/src/models/mockJobDataObjects.js]: changed

**Review by JudeShamsi — APPROVED** [review]

**Pull request merged**